gremlin-server/src/main/java/org/apache/tinkerpop/gremlin/server/handler/WsAndHttpChannelizerHandler.java

```diff
@@ -100,6 +105,12 @@ public void channelRead(final ChannelHandlerContext ctx, final Object obj) {
 
     @Override
     public void exceptionCaught(final ChannelHandlerContext ctx, final Throwable cause) {
+        // The log printing here is necessary because when an exception thrown by Netty reaches this method,
+        // if the error is not printed, the server side will not be able to perceive what has happened.
+        // For example, if the maxContentLength in JanusGraph's Gremlin-Server configuration is set to 65536,
+        // and the client sends data over this size via WebSocket, the JanusGraph server logs will appear normal,
+        // but the client will experience a disconnection, making it difficult to immediately identify the cause of the issue.
+        logger.error("Error processing request from netty", cause);
```

Comment:
```suggestion
        logger.error("Could not process the request", cause);
```
Nit: the error should be slightly more generic to match what is done in `gremlin-driver`